apps/overige-objecten-api/src/utils/getVacData copy.ts

```diff
@@ -0,0 +1,52 @@
+import kebabCase from 'lodash.kebabcase';
+import { concatenateFieldValues } from './concatenateFieldValues';
+import type { Antwoord, DataVacItem, VACSData } from '../strapi-product-type';
+
+interface GetVacDataProps {
+  data: VACSData;
+  vacSchemaURL: string;
+  serverURL: string;
+}
+interface MapVAcItemProps extends Pick<GetVacDataProps, 'serverURL' | 'vacSchemaURL'> {
```

Comment:
```suggestion
interface MapVacItemProps extends Pick<GetVacDataProps, 'serverURL' | 'vacSchemaURL'> {
```

en natuurlijk ook op de andere plekken aanpassen